packages/form-validation/src/validation.js

```diff
@@ -26,10 +26,14 @@ export default class AuroFormValidation {
         elem.validity = 'badInput';
         elem.setCustomValidity = elem.setCustomValidityBadInput || '';
       }
-    } else if (elem.value && elem.value.length > 0 && elem.value.length < elem.minLength) {
+    }
+
+    if (elem.value && elem.value.length > 0 && elem.value.length < elem.minLength) {
       elem.validity = 'tooShort';
       elem.setCustomValidity = elem.setCustomValidityTooShort || '';
-    } else if (elem.value && elem.value.length > elem.maxLength) {
+    }
```

Comment:
Could this be condensed to:

```
if (elem.value?.length < elem.minLength) {
  elem.validity = 'tooShort';
  elem.setCustomValidity = elem.setCustomValidityTooShort || '';
}
```

Reply:
If I remember correctly, we at least need to keep the `length > 0` part in order for this error not to be triggered when a user simply just interacts with the element.

Reply:
Should that be left as a code comment?